comps/llms/text-generation/native/docker/Dockerfile

```diff
@@ -11,13 +9,11 @@ ENV LANG=en_US.UTF-8
 ARG REPO=https://github.com/huggingface/optimum-habana.git
 ARG REPO_VER=v1.12.1
 
-RUN apt-get update && \
-    apt-get install git-lfs && \
-    git-lfs install && \
-    apt-get install -y --no-install-recommends --fix-missing \
+RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing \
+    git-lfs \
     libgl1-mesa-glx \
-    libjemalloc-dev \
-    vim
+    libjemalloc-dev && \
+    git-lfs install
```

Comment:
Do we need this separate install?

Reply:
Yes, we do need it because it's actually installing it after download.

Reply:
But on a second thought, `git lfs` should be initialized per user which means it needs to be done during `user`'s session rather than root and I fixed that.